webpack.prod.js

```diff
@@ -0,0 +1,16 @@
+const path = require('path');
+const {merge} = require('webpack-merge');
+const base = require('./webpack.config');
+
+module.exports = merge(base, {
+    
+    mode: 'production',
+
+    devtool: 'source-map',
+
+    output: {
+        path: path.resolve(__dirname, './distProd'),
+        filename: 'prod.bundle.js',
+    },
```

Comment:
```suggestion
    output: {
        path: path.resolve(__dirname, './dist'),
        filename: 'prod.bundle.js',
    },
```
Сделай пожалуйста для дев и прод одну папку билда и помести `output` в базовый конфиг